docs/src/pages/components/material-icons/synonyms.js

```diff
@@ -349,6 +351,371 @@ const synonyms = {
+  IcMarkunreadMailbox: 'deliver envelop letter postal postbox receive send',
+  IcMaximize: 'line shape',
+  IcMessage: 'bubble chat speech talk text',
+  IcMic: 'hearing microphone noise record sound voice',
+  IcMicNone: 'hearing microphone noise record sound voice',
+  IcMicOff: 'hearing microphone noise record sound voice',
+  IcMinimize: 'line shape',
+  IcMissedVideoCall: 'arrow record',
+  IcMobileScreenShare: 'arrow device mirror monitor phone tv',
+  IcMotorcycle: 'bicycle bike moped ride riding transportation vehicle wheels',
+  IcMovie: 'screen show tv watch',
+  IcMusicVideo: 'band recording screen tv watch',
+  IcNewReleases: 'alert announcement burst exclamation star',
+  IcNoSim: 'camera card device eject insert memory phone storage',
+  IcNote: 'bookmark message paper',
+  IcNotificationImportant: 'alert announcement bell feedback problem',
+  IcNotInterested: 'circle dislike interested not off x',
+  IcOfflineBolt: 'circle flash lightning spark',
+  IcOpacity: 'color droplet hue inverted palette tone water',
+  IcOpenInBrowser: 'arrow box new window',
+  IcOpenInNew: 'arrow box window',
+  IcOpenWith: 'arrows direction expand move',
+  IcPageview: 'document find glass magnifying paper search',
+  IcPanTool: 'hand human move scan stop',
+  IcPause: 'controls music video',
+  IcPauseCircleFilled: 'controls music video',
+  IcPauseCircleOutline: 'controls music video',
+  IcPausePresentation: 'device screen share',
+  IcPayment: 'card charge credit information money',
+  IcPermCameraMic: 'microphone photo speaker',
+  IcPermContactCalendar: 'human person profile',
+  IcPermDataSetting: 'information settings',
+  IcPermDeviceInformation: 'alert announcement cell important phone',
+  IcPermIdentity: 'human information person profile save,',
+  IcPermMedia: 'collection folders images mountain save',
+  IcPermPhoneMsg: 'bubble chat message recording save speech voice',
+  IcPermScanWifi: 'alert announcement information internet network service',
+  IcPersonAddDisabled: 'profile user',
+  IcPets: 'animal dog hand paw',
+  IcPhone: 'call chat device mobile text',
+  IcPhonelinkErase: 'device mobile',
+  IcPhonelinkLock: 'device mobile',
+  IcPhonelinkRing: 'device mobile',
+  IcPhonelinkSetup: 'device information mobile settings',
+  IcPictureInPicture: 'cropped overlap photo shape',
+  IcPictureInPictureAlt: 'cropped overlap photo shape',
+  IcPlayArrow: 'controls music video',
+  IcPlayCircleFilled: 'arrow controls music video',
+  IcPlayCircleOutline: 'arrow controls music video',
+  IcPlayForWork: 'arrow circle down google half',
+  IcPlaylistAdd: 'collection music plus',
+  IcPlaylistAddCheck: 'collection music',
+  IcPlaylistPlay: 'collection music',
+  IcPolymer: 'emblem logo mark',
+  IcPortableWifiOff: 'connected connection device internet service usage',
+  IcPowerSettingsNew: 'information off save shutdown',
+  IcPregnantWoman: 'baby birth female human lady maternity person',
+  IcPrint: 'draft ink paper printer send',
```

Comment:
strange, I'm not aware that we such icon.

Reply:
Yeah, I only spotted it in the diff. I've updated the script.